src/trigger/text_area.hpp

```diff
@@ -54,6 +54,7 @@ class TextArea final : public TriggerBase
   size_t m_current_text;
   Status m_status;
   Timer m_timer;
+  int m_anchor;
 
```

Comment:
In reference to above, this variable can be declared with type `AnchorPoint`, which is more logical to use and is more verbose; it'll help future maintainers to easily understand the variable.